src/ray/common/scheduling/scheduling_test.cc

```diff
@@ -5,8 +5,106 @@
 #include <thread>
 
 #include "ray/common/scheduling/scheduling_ids.h"
+#include "ray/common/scheduling/scheduling_alg.h"
 using namespace std;
 
+
+void initTaskReq(TaskReq &tr,
```

Comment:
please use vectors instead of raw pointers - this looks very much like C code :)

Reply:
Done